src/components/ConversionServerErrorModal/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from "antd";
+import React from "react";
+
+import CustomModal from "../CustomModal";
+import { Exclamation } from "../Icons";
+
+import styles from "./style.css";
+
+interface ConversionServerErrorModalProps {
+    closeModal: () => void;
+}
+
+const ConversionServerErrorModal: React.FC<ConversionServerErrorModalProps> = ({
+    closeModal,
+}) => {
+    return (
+        <CustomModal
+            className={styles.serverErrorModal}
+            title="File import cannot be completed"
+            open
+            footer={
+                <Button type="primary" onClick={closeModal}>
+                    OK
+                </Button>
+            }
+            centered
+            onCancel={closeModal}
+        >
+            <p className={styles.warningText}>
+                {" "}
+                {Exclamation} We&apos;re sorry, the server is currently
+                experiencing an issue.
+            </p>
+            <p>
+                {" "}
+                Please try again at a later time. For further assistance, please
+                visit{" "}
```

Comment:
This one is the only one that I think you need?

Reply:
i think i can remove them all and it looks fine